index.html

```diff
@@ -0,0 +1,101 @@
+        </section>
+
+        <section class="program-section">
+            <h2 class="program-title">Main Program</h2>
+            <div class="red-border"></div>
+            <div class="program-cards">
+                <div class="cards card-1">
+                    <img class="cards-img" src="./images/icons/program_icon_01.png" alt="icon1">
+                    <h3>Disney</h3>
+                    <p>The Walt Disney Co. is a diversified international family entertainment and media enterprise.</p>
+                </div>
+                <div class="cards card-2">
+                    <img class="cards-img" src="./images/icons/program_icon_02.png" alt="icon2">
+                    <h3>Movies</h3>
+                    <p>You can have a whole life experience by watching all our brand new movies. Also, don't forget to check the classic ones!</p>
+                </div>
+                <div class="cards card-3">
+                    <img class="cards-img" src="./images/icons/program_icon_03.png" alt="icon3">
+                    <h3>Tickets</h3>
+                    <p>Come join us on a magical trip on the Walt Disney Kingdom park.</p>
+                </div>
+                <div class="cards card-4">
+                    <img class="cards-img" src="./images/icons/program_icon_04.png" alt="icon4">
+                    <h3>Shop</h3>
+                    <p>Stay connected to your favorite characters with these Toy's collection.</p>
+                </div>
+                <div class="cards card-5">
+                    <img class="cards-img" src="./images/icons/program_icon_05.png" alt="icon5">
+                    <h3>Books</h3>
+                    <p>This is where the magic starts. Check our store for all the Disney World book's collection.</p>
+                </div>
+            </div>
+            <button class="program-button">Join this Experience 2022</button>
+        </section>
+
+        <section class="speakers-section">
+            <h2 class="speakers-title">Featured Characters</h2>
+            <div class="red-border"></div>
+            <div class="speakers">
+                <!--generating speakers dinamically-->
+            </div>
+        </section>
+
+        <section class="partners-section">
+            <h2 class="partners-title">Main Sponsors</h2>
+            <div class="red-border"></div>
+            <div class="partners-div">
+                <img class="logo1" src="./images/Partners/mozilla-logo.png" alt="mozilla-logo">
+                <img class="logo2" src="./images/Partners/google-logo.png" alt="google-logo">
+            </div>
+        </section>
+    </main>
+
+    <footer class="footer-section">
+        <div class="footer-logo">
+            <img class="footer-logo-img" src="./images/Partners/disney.png" alt="footer-image">
+        </div>
+
+        <div class="footer-texts">
+            <p>2022 Creative Commons Orlando.</p>
+            <p>Some Rights Reserved.</p>
+        </div>
+    </footer>
```

Comment:
- [ ] Kindly add these two sections to the mobile version too on the Home page, I know they are not showing on the original design, but all sections on the desktop commonly appear on the mobile version. it just doesn't appear in the pic.

![image](https://user-images.githubusercontent.com/35535130/174897108-dc93011c-7bf1-4b0a-ac52-726bd428cfa3.png)

Reply:
Implemented 👍

Reply:
Done! ✔️